templates/admin/report/show_details.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends '@SonataAdmin/CRUD/base_show_field.html.twig' %}
+
+{% block field_value %}
+    {% if object.subject is instanceof('AppBundle\\Entity\\EntitySoftDeletedInterface') and object.subject.getDeletedAt() is not null %}
```

Comment:
what about including `partial/_details.html.twig` here to avoid duplicate code and fix the little bug with `(dés)approuvé` label?

Reply:
It's not a bug. On the report show page we need show information only if an reported object is approved and on the thread/comment page the both status should be shown.